src/ui/components/CollectionFilters/CollectionFilters.tsx

```diff
@@ -26,6 +26,10 @@ export enum SortType {
 }
 
 export const collectionPageViewModeStore = 'collectionPageViewMode';
+export const collectionPageFilterOnlyMyStore = 'collectionPageFilterOnlyMyStore';
```

Comment:
Can we store all filters as JSON in one localStorage field – `collectionPageFilters`?

Reply:
good idea, done